server/remote/gitea/gitea.go

```diff
@@ -540,3 +557,39 @@ func getStatus(status model.StatusValue) gitea.StatusState {
 		return gitea.StatusFailure
 	}
 }
+
+func (c *Gitea) getChangedFilesForPR(ctx context.Context, repo *model.Repo, index int64) ([]string, error) {
+	_store, ok := store.TryFromContext(ctx)
+	if !ok {
+		log.Error().Msg("could not get store from context")
+		return []string{}, nil
+	}
+
+	repo, err := _store.GetRepoNameFallback(repo.RemoteID, repo.FullName)
+	if err != nil {
+		return nil, err
+	}
+
+	user, err := _store.GetUser(repo.UserID)
+	if err != nil {
+		return nil, err
+	}
+
+	client, err := c.newClientToken(ctx, user.Token)
+	if err != nil {
+		return nil, err
+	}
+	return common.Paginate(func(page int) ([]string, error) {
```

Comment:
check if the error is "to low gitea version" and return no errror an no changed files -> as we have it now

Reply:
asap we have init the client we can do ... but in terms of api cals there wont be any difference

Reply:
(the sdk do check on it's own before calling the endpoint)

Reply:
It's done on client init if you dont tell to do otherwhise